docs/download.md

```diff
@@ -21,14 +21,12 @@ license: |
 
 No version avaiable yet.
 
-<!--
 ### 0.2.0-incubating (2022-12-15)
```

Comment:
Need update the date?

Reply:
This should be the ANNOUNCE date, thanks for catching